watertap/examples/flowsheets/RO_with_energy_recovery/RO_with_energy_recovery.py

```diff
@@ -350,7 +382,7 @@ def initialize_system(m, solver=None):
     optarg = solver.options
 
     # ---initialize RO---
-    m.fs.RO.initialize(optarg=optarg)
+    m.fs.RO.initialize(optarg=optarg, outlvl=idaeslog.DEBUG)
```

Comment:
We can probably take away this DEBUG setting to minimize logger output by default